Automattic/wp-calypso
# Switch from useIsSimpleSeller hook to getSiteOption

#### Changes proposed in this Pull Request

The hook isn't necessary. All we need to do is use `getSiteOption` to check if the site intent is `sell` on the WooCommerce landing page (the only place the hook was being used).

#### Testing instructions

Visit a non-seller website admin and go to `/woocommerce-installation`. You should see the following content:

![image](https://user-images.githubusercontent.com/917632/157089939-dd1d7120-6236-42b7-abd9-dddd101fe5f6.png)

Then visit the `/woocommerce-installation` page for a website that used the "Start simple" seller signup flow. You should see the following content instead:

![image](https://user-images.githubusercontent.com/917632/157090028-846fadd7-5e1e-4a10-8394-68e1d2875eee.png)

Related to #62031
